dev/integration_tests/flutter_gallery/pubspec.yaml

```diff
@@ -11,10 +11,10 @@ dependencies:
   intl: 0.18.1
   connectivity: 3.0.6
   string_scanner: 1.2.0
-  url_launcher: 6.1.12
+  url_launcher: 6.1.13
   # This is listed as direct so it can be manually pinned
-  url_launcher_android: 6.0.17
-  cupertino_icons: 1.0.5
+  url_launcher_android: 6.0.39
```

Comment:
This got updated correctly, but I'll push another commit after deleting it and re-running `flutter update-packages --force-upgrade` so that it is properly marked with `THIS LINE IS AUTOGENERATED`